database/migrations/2024_06_20_171822_create_airline_city_table.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('airline_city', function (Blueprint $table) {
+            $table->id();
+            $table->foreignIdFor(\App\Models\Airline::class, 'airline_id')->constrained()->cascadeOnDelete();
+            $table->foreignIdFor(\App\Models\City::class, 'city_id')->constrained()->cascadeOnDelete();
```

Comment:
```suggestion
            $table
                ->foreignIdFor(Airline::class)
                ->constrained()
                ->cascadeOnDelete();
            $table
                ->foreignIdFor(City::class)
                ->constrained()
                ->cascadeOnDelete();
```
In this case, you can skip passing the column name to the `foreignIdFor` because Laravel will create the column based on the lowercase-snake-case model + "_id". It makes more sense to pass the column name when you want to avoid Laravel creating the column based on the model's name.

On a different note, always import classes/interfaces/stuff.